lib/hanami/assets/helpers.rb

```diff
@@ -682,6 +752,15 @@ def _typed_asset_path(source, ext)
         asset_path(source)
       end
 
+      def _sri?
+        !!self.class.assets_configuration.sri
+      end
+
+      def _sri_value(source, ext)
+        source = "#{ source }#{ ext }" unless source.match(/#{ Regexp.escape(ext) }\z/)
+        self.class.assets_configuration.sri_value(source) unless _absolute_url?(source)
```

Comment:
@jodosha `:sri` is the configuration option, `:_sri?` is a utility method that determines if SRI is enabled, and `:integrity` is the option passed to the `javascript` method that specifies an SRI digest hash for the script (of stylesheet). So, yes, you're right. :smile: